andela/reactors-cards-for-humanity
# #143412465 Re-design the various gaming screens

#### What does this PR do?
- Changes the design of the screen a user sees when playing the game
 #### Description of Task to be completed?
- Re-design the screens a user interacts with when playing the game in tandem with the new theme.
 #### How should this be manually tested?
- run `npm start`
- Click on the ```Start Game``` button
- Select the `Play with Friends` option
- Copy the link given and copy it.
- Paste the ink in multiple tabs (to simulate other players joining the game)
- Navigate to the original page and click on the `Start game with _ players` (where the _ represents the number of tabs opened)
- This should enable interaction with the gaming screens.
 #### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/143412465
 #### Screenshots (if appropriate)
<img width="1432" alt="screenshot 2017-05-16 15 55 43" src="https://cloud.githubusercontent.com/assets/27014312/26106972/76ed63f6-3a50-11e7-8c04-2d8f505eeb7c.png">
<img width="1435" alt="screenshot 2017-05-16 15 57 20" src="https://cloud.githubusercontent.com/assets/27014312/26106980/7cc45460-3a50-11e7-8438-a4e20ff17ea3.png">
<img width="1437" alt="screenshot 2017-05-16 15 57 45" src="https://cloud.githubusercontent.com/assets/27014312/26106985/7e8d13fe-3a50-11e7-92cd-09b7e2a5e220.png">
